silkworm/rpc/json/stream.hpp

```diff
@@ -44,9 +44,7 @@ struct DataChunk {
 //! Stream can be used to send big JSON data split into multiple fragments.
 class Stream {
   public:
-    inline static constexpr std::size_t kDefaultCapacity{5 * 1024 * 1024};
-
-    Stream(boost::asio::any_io_executor& executor, StreamWriter& writer, std::size_t buffer_capacity = kDefaultCapacity);
+    Stream(boost::asio::any_io_executor& executor, StreamWriter& writer, size_t new_buffer_capacity = 0);
```

Comment:
`new_buffer_capacity` -> `buffer_capacity`

Moreover, I suggest defaulting `buffer_capacity` value to `writer_.get_capacity()` directly here